src/main/resources/pdf_templates/ds01.html

```diff
@@ -0,0 +1,209 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en">
+
+<head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type"/>
+    <title>DS01</title>
+
+    <style type="text/css">
+        body {
+            font-size: 14px;
+            font-family: 'TransportNewLight-Regular', sans-serif;
+        }
+
+        img {
+            height: 100%;
+            width: 100%;
+        }
+
+        #page-container {
+            width: 600px;
+            margin: 0 auto
+        }
+
+        .upper {
+            height: 280px;
+            float: left;
```

Comment:
I think your use of float could be causing the accessibility issues. Why do you need float? You aren't trying to shift anything to the left. Isn't HTML on the left by default? It should be, unless you have added some styles that cause it not to be.

try removing these floats and see id this changes the PAVE reading.

Reply:
It was justify